libs/datasets/vaccine_backfills.py

```diff
@@ -18,36 +16,33 @@ def derive_vaccine_pct(ds_in: MultiRegionDataset) -> MultiRegionDataset:
         CommonFields.VACCINATIONS_INITIATED: CommonFields.VACCINATIONS_INITIATED_PCT,
         CommonFields.VACCINATIONS_COMPLETED: CommonFields.VACCINATIONS_COMPLETED_PCT,
     }
-    ds_in_wide_dates = ds_in.timeseries_not_bucketed_wide_dates
-    ds_in_wide_dates = ds_in_wide_dates.loc[
-        ds_in_wide_dates.index.get_level_values(PdFields.VARIABLE).isin(field_map.keys())
-    ]
+
+    ts_in_all = ds_in.timeseries_bucketed_wide_dates
+    ts_in_all_variable_index = ts_in_all.index.get_level_values(PdFields.VARIABLE)
+    ts_counts = ts_in_all.loc(axis=0)[ts_in_all_variable_index.isin(field_map.keys())]
```

Comment:
Consider `ts_counts` => `ts_in_cumulatives` or `ts_in_people` or something?

Also, sorry, can you explain what `axis=0` is for here?  It's not documented in the [loc](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.loc.html) docs and the tests pass with or without it.  Is it actually doing something meaningful?

Reply:
Rename done. I learned about `loc(axis=0)` on https://pandas.pydata.org/docs/user_guide/advanced.html and thought it is nice to differentiate between df.loc[row, column] df.loc[row_level0, row_level1], and df.loc[row]. Removed because I wasn't even consistent about using it in this function.